spec/system/polis_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
+
+  let(:site_id) { ENV["POLIS_SITE_ID"] } # to store in local
```

Comment:
Please remove useless comment
```suggestion
  let(:site_id) { ENV["POLIS_SITE_ID"] }
```